productsapp/components/Login.js

```diff
@@ -1,54 +1,98 @@
 import * as React from 'react';
-import { Text, View, TextInput, Image, TouchableOpacity } from 'react-native';
-import ProductsList from '/ProductsList';
+import {
+  Text,
+  View,
+  TextInput,
+  ActivityIndicator,
+  Image,
+  FlatList,
+  TouchableOpacity,
+  Alert,
+} from 'react-native';
 import { Font } from 'expo';
 import styles from './Styles';
-
+import ProductsList from './ProductsList';
 export default class Login extends React.Component<{}> {
-   componentDidMount() {
+  constructor(props) {
+    super(props);
+    this.state = { isLoading: true };
+    this.state = { username: '' };
+    this.state = { password: '' };
+    this.state = { fontLoaded: false };
+  }
+
+  componentWillMount() {
     Font.loadAsync({
-      'vincHand': require('../assets/fonts/VINCHAND.ttf'),
+      vincHand: require('../assets/fonts/VINCHAND.ttf'),
     });
+    this.setState({ fontLoaded: true });
   }
-  constructor(props){
-    super(props)
-    this.state = { showResults: true };
-  }
-
-  showProductsList = () => {
-       this.setState({ showResults: false })
-   }
+  handleClick = () => {
+    return fetch(
+      'http://ecsc00a02fb3.epam.com/index.php/rest/V1/integration/customer/token',
+      {
+        method: 'POST',
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          username: this.state.username,
+          password: this.state.password,
+        }),
+      }
+    )
+      .then(response => response.json())
+      .then(responseJson => {
```

Comment:
auth don't work at all =/
server always return responseJson both in success and in failure case, and all types of credentials a valid in your app. 
Error thrown only in cause of network problems.
To set setup auth process correct you may check errors messages inside of responseJson object